src/style/internal/lime-theme.scss

```diff
@@ -7,41 +7,7 @@
  * Avoid that unless there's very good reason for it!
  */
 
-@use '../brand-colors';
 @use '../_theme-color-variables';
```

Comment:
it affected limebb-navigation-button 
now:
<img width="1096" height="614" alt="Screenshot 2025-08-04 at 14 10 12" src="https://github.com/user-attachments/assets/fe052079-5eee-4c8b-a536-a35acd63696b" />
before:

<img width="1059" height="1187" alt="Screenshot 2025-08-04 at 14 10 23" src="https://github.com/user-attachments/assets/34e22d62-06dc-484f-87ec-9f8d2133af59" />